src/cgr_gwas_qc/workflow/scripts/vcf2adpc.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
```

Comment:
remove commented line 4

Reply:
Removed in ffdcf3fe68e81fc6e28fd295d969c81a0cc28fdd.